benchmarks/benchmarks/stats.py

```diff
@@ -56,7 +56,97 @@ def time_ttest_ind_diff_var(self):
         stats.ttest_ind(self.a, self.c, equal_var=False)
 
 
+class DistributionsAll(Benchmark):
+    # all distributions are in this list. A conversion to a set is used to
+    # remove duplicates that appear twince in either `distcont` or
+    # `distdiscrete`.
+    dists = sorted(list(set([d[0] for d in distcont + distdiscrete])))
+
+    param_names = ['distribution', 'method']
+    params = [
+        dists, ['pdf/pmf', 'logpdf/logpmf', 'cdf', 'logcdf', 'rvs', 'fit',
+                'sf', 'logsf', 'ppf', 'isf', 'moment', 'stats_s', 'stats_v',
+                'stats_m', 'stats_k', 'stats_mvsk', 'entropy']
+    ]
+    dist_data = dict(distcont + distdiscrete)
+    # custom shape values can be provided for any distribution in the format
+    # `dist_name`: [shape1, shape2, ...]
+    custom_input = {}
+
+    # these are the distributions that are the slowest
+    slow_dists = ['nct', 'ncx2', 'argus', 'cosine', 'foldnorm', 'gausshyper',
+                  'kappa4', 'invgauss', 'wald', 'vonmises_line', 'ksone',
+                  'genexpon', 'exponnorm', 'recipinvgauss', 'vonmises',
+                  'foldcauchy', 'kstwo', 'levy_stable', 'skewnorm']
+    slow_methods = ['moment']
+
+    def setup(self, distribution, method):
+        if not is_xslow() and (distribution in self.slow_dists or
+                               method in self.slow_methods):
```

Comment:
```suggestion
        if not is_xslow() and (distribution in self.slow_dists 
                               or method in self.slow_methods):
```
ideally binary operators go at beginning of continuation line per PEP8